test/cpp/qps/perf_db.proto

```diff
@@ -29,7 +29,7 @@
 
 syntax = "proto3";
```

Comment:
Just confirming that perf_db is only used by the driver as an alternate reporting mechanism, not by the qps_worker (or any other solo client or server), so this should be fine. Will reconsider if needed.